internal/config/cgmgr/cgroupfs_linux.go

```diff
@@ -90,25 +90,25 @@ func (m *CgroupfsManager) ContainerCgroupStats(sbParent, containerID string) (cg
 	}
 	cgstats = &CgroupStats{
 		MostStats:     libCtrStats,
-		OtherMemStats: generateOtherMemoryStats(libCtrStats.MemoryStats),
+		OtherMemStats: generateOtherMemoryStats(&libCtrStats.MemoryStats),
 		SystemNano:    time.Now().UnixNano(),
 	}
 	return cgstats, nil
 }
 
 // GetCtrCgroupManager takes the cgroup parent, and container ID.
 // It returns the raw libcontainer cgroup manager for that container.
-func (m *CgroupfsManager) GetCtrCgroupManager(sbParent, containerID string) (cgMgr libctrCg.Manager, err error) {
+func (m *CgroupfsManager) GetCtrCgroupManager(sbParent, containerID string) (libctrCg.Manager, error) {
```

Comment:
Naming suggestion: golang sugggests not prepending `Get` to getters. Also ,the other functions in this package spell out Container, so maybe `ContainerCgroupManager` ?